dask_ml/preprocessing/label.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import division
+
+from operator import getitem
+
+import dask.array as da
+import dask.dataframe as dd
+import numpy as np
+from sklearn.preprocessing import label as sklabel
+from sklearn.utils.validation import check_is_fitted
+
+
+class LabelEncoder(sklabel.LabelEncoder):
+
+    __doc__ = sklabel.LabelEncoder.__doc__
+
+    def fit(self, y):
+        if isinstance(y, dd.Series):
```

Comment:
Slight preference for moving this to a `LabelEnoder._check_array`. Then it can be re-used with `LabelEncodertransform`.

Reply:
Can do @TomAugspurger, that seems reasonable to me